cloudflare_worker/worker/rollup.config.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import glob from 'glob';
+import typescript from '@rollup/plugin-typescript';
+
+const testSrcFiles = glob.sync('src/**/*.test.ts');
+
+export default [
+  {
+    input: 'src/index.ts',
+    output: {
+      dir: 'dist',
+      format: 'es'
```

Comment:
Maybe add a short comment saying what 'es' is. (Ditto 'cjs' below.)

Reply:
Comment added. The format is now changed to the more appropriate format 'iife'.